### PR TITLE
Add elixir contracts ci

### PR DESCRIPTION
**Pull request by rtorrero**
This PR follows on the work from pr #5 to include also elixir contracts generation checks

Again, it:

- Adds validation: it will cause a broken / invalid schema to fail the pipeline, preventing us from merging a broken PR
- Checks for uncommited contracts: If someone forgets to add the generated go contracts to the commit, the pipeline will fail

### CODE REVIEW

**Review by arbulu89 — APPROVED** [review]
Looks good to me!
Maybe there was a better way to use the `clean` in the main makefile, but it is fine as it is hehe

**Comment by rtorrero** [comment]
Thanks to @arbulu89 for pairing with me on this

File: elixir/Makefile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.PHONY: all clean elixir-generate
+SCHEMAS = ../schema
+
+all: elixir-generate
+
+elixir-generate: $(SCHEMAS)/*.json
+	mix deps.get
+	mix contracts.generate
```
Comment (nelsonkopliku) [inline elixir/Makefile:8]: Does this need a format as well?

**Review by nelsonkopliku — APPROVED** [review]
LGTM

File: elixir/Makefile (hunk 1 above)
Comment (rtorrero) [reply elixir/Makefile:8]: Ouch :-) Good catch...

**Review by CDimonaco — APPROVED** [review]

**Pull request merged**